docs/Examples/BbhInitialData/BbhInitialData.ipynb

```diff
@@ -0,0 +1,463 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Distributed under the MIT License.\n",
+    "# See LICENSE.txt for details.\n",
+    "\n",
+    "import os\n",
+    "import h5py\n",
+    "import pandas as pd\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "from ruamel.yaml import YAML"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "# Clean up output files from previous runs\n",
+    "rm -f Bbh*.h5"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Binary black hole initial data\n",
+    "\n",
+    "In this example we run the elliptic solver to compute initial data for binary\n",
+    "black holes."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, compile the `SolveXcts` executable and put in the path to your build\n",
+    "directory below. Make sure you compile in `Release` mode."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "SPECTRE_BUILD_DIR = \"/work/nfischer/spectre/build-Default-Release/\"\n",
+    "SPECTRE_HOME = os.path.abspath(\"../../../\")"
```

Comment:
I'd also make this an absolute path. Maybe make this work in the container and use paths there?

Reply:
I adjusted things a bit so the notebook runs in the container. I think once we have developed our Python package a bit further we may be able to remove at least some of these hard-coded paths, because the Python package will be able to at least point to the executables that are installed alongside of it (with the option to use another user-specified build directory if desired). The test input files and/or examples we may also copy to the build dir and the install prefix in the future, but let's not decide these things in this PR :) For now I put two absolute paths here that the user will have to set.

Reply:
Agreed, that'll make things easier :)